inlanecruising_plugin/src/inlanecruising_plugin.cpp

```diff
@@ -267,4 +267,89 @@ namespace inlanecruising_plugin
 
         return trajectory;
     }
+
+    tk::spline InLaneCruisingPlugin::compute_fit(std::vector<lanelet::BasicPoint2d> basic_points){
+        int n = int(basic_points.size())/3;
```

Comment:
The int() here is not needed as all types here are already integers and I don't think will compile. A cast is normally shown as (int) not int()